### PR TITLE
Commercial switch tidy up

### PR DESCRIPTION
**Pull request by rich-nguyen**
Removing unused/uneeded commercial switches, @SiAdcock

### CODE REVIEW

**Comment by rich-nguyen** [comment]
[switches tidy up](https://trello.com/c/p0zdUsHf/114-switches-tidy-up)

File: admin/app/dfp/DataAgent.scala [hunk 1]
```diff
@@ -18,24 +18,19 @@ trait DataAgent[K, V] extends ExecutionContexts with Logging with implicits.Stri
...
-              DataCache(freshData)
-            } else {
-              log.error("No fresh data loaded so keeping old data")
-              oldCache
-            }
-          case Failure(e) =>
-            log.error(ExceptionUtils.getStackTrace(e))
+      loadFreshData() match {
+        case Success(freshData) =>
+          if (freshData.nonEmpty) {
```
Comment (janua) [inline admin/app/dfp/DataAgent.scala]: Cleaner with a guard?

**Review by regiskuckaertz — APPROVED** [review]
yiihaa aside from @janua 's question

File: admin/app/dfp/DataAgent.scala (hunk 1 above)
Comment (rich-nguyen) [inline admin/app/dfp/DataAgent.scala]: yeah, I just unwrapped the switch block

**Comment by janua** [comment]
LGTM :+1:

**Pull request merged**